diff --git a/notifier/dbus.c b/notifier/dbus.c
index 1ebd2ea..7289862 100644
--- a/notifier/dbus.c
+++ b/notifier/dbus.c
@@ -256,19 +256,19 @@ main(int  argc,				/* I - Number of command-line args */
 
     if (con && !dbus_connection_get_is_connected(con))
     {
-      dbus_connection_unref(con);
+//      dbus_connection_unref(con);
       con = NULL;
     }
 
     if (!con)
     {
-      dbus_error_init(&error);
+//      dbus_error_init(&error);
 
-      con = dbus_bus_get(DBUS_BUS_SYSTEM, &error);
-      if (!con)
-	dbus_error_free(&error);
-      else
-	fputs("DEBUG: Connected to D-BUS\n", stderr);
+//      con = dbus_bus_get(DBUS_BUS_SYSTEM, &error);
+//      if (!con)
+//	dbus_error_free(&error);
+//      else
+//	fputs("DEBUG: Connected to D-BUS\n", stderr);
     }
 
     if (!con)
@@ -353,17 +353,17 @@ main(int  argc,				/* I - Number of command-line args */
      */
 
     fprintf(stderr, "DEBUG: %s\n", signame);
-    message = dbus_message_new_signal("/org/cups/cupsd/Notifier",
-				      "org.cups.cupsd.Notifier",
-				      signame);
+//    message = dbus_message_new_signal("/org/cups/cupsd/Notifier",
+//				      "org.cups.cupsd.Notifier",
+//				      signame);
 
-    dbus_message_append_iter_init(message, &iter);
+//    dbus_message_append_iter_init(message, &iter);
     attr = ippFindAttribute(msg, "notify-text", IPP_TAG_TEXT);
     if (attr)
     {
       const char *val = ippGetString(attr, 0, NULL);
-      if (!dbus_message_iter_append_string(&iter, &val))
-        goto bail;
+//      if (!dbus_message_iter_append_string(&iter, &val))
+//        goto bail;
     }
     else
       goto bail;
@@ -380,13 +380,13 @@ main(int  argc,				/* I - Number of command-line args */
       if (attr)
       {
         const char *val = ippGetString(attr, 0, NULL);
-        if (!dbus_message_iter_append_string(&iter, &val))
-	  goto bail;
+//        if (!dbus_message_iter_append_string(&iter, &val))
+//	  goto bail;
       }
       else
       {
 	have_printer_params = 0;
-	dbus_message_iter_append_string(&iter, &nul);
+//	dbus_message_iter_append_string(&iter, &nul);
       }
 
       /* STRING printer-name */
@@ -396,14 +396,14 @@ main(int  argc,				/* I - Number of command-line args */
         if (attr)
         {
           const char *val = ippGetString(attr, 0, NULL);
-          if (!dbus_message_iter_append_string(&iter, &val))
-            goto bail;
+//          if (!dbus_message_iter_append_string(&iter, &val))
+//            goto bail;
         }
         else
           goto bail;
       }
-      else
-	dbus_message_iter_append_string(&iter, &nul);
+//      else
+//	dbus_message_iter_append_string(&iter, &nul);
 
       /* UINT32 printer-state */
       if (have_printer_params)
@@ -412,13 +412,13 @@ main(int  argc,				/* I - Number of command-line args */
 	if (attr)
 	{
 	  dbus_uint32_t val = (dbus_uint32_t)ippGetInteger(attr, 0);
-	  dbus_message_iter_append_uint32(&iter, &val);
+//	  dbus_message_iter_append_uint32(&iter, &val);
 	}
 	else
 	  goto bail;
       }
-      else
-	dbus_message_iter_append_uint32(&iter, &no);
+//      else
+//	dbus_message_iter_append_uint32(&iter, &no);
 
       /* STRING printer-state-reasons */
       if (have_printer_params)
@@ -443,14 +443,14 @@ main(int  argc,				/* I - Number of command-line args */
 	    strlcpy(p, ippGetString(attr, i, NULL), reasons_length - (size_t)(p - printer_reasons));
 	    p += strlen(p);
 	  }
-	  if (!dbus_message_iter_append_string(&iter, &printer_reasons))
-	    goto bail;
+//	  if (!dbus_message_iter_append_string(&iter, &printer_reasons))
+//	    goto bail;
 	}
 	else
 	  goto bail;
       }
-      else
-	dbus_message_iter_append_string(&iter, &nul);
+//      else
+//	dbus_message_iter_append_string(&iter, &nul);
 
       /* BOOL printer-is-accepting-jobs */
       if (have_printer_params)
@@ -460,13 +460,13 @@ main(int  argc,				/* I - Number of command-line args */
 	if (attr)
 	{
 	  dbus_bool_t val = (dbus_bool_t)ippGetBoolean(attr, 0);
-	  dbus_message_iter_append_boolean(&iter, &val);
+//	  dbus_message_iter_append_boolean(&iter, &val);
 	}
 	else
 	  goto bail;
       }
-      else
-	dbus_message_iter_append_boolean(&iter, &no);
+//      else
+//	dbus_message_iter_append_boolean(&iter, &no);
     }
 
     if (params >= PARAMS_JOB)
@@ -480,7 +480,7 @@ main(int  argc,				/* I - Number of command-line args */
       if (attr)
       {
         dbus_uint32_t val = (dbus_uint32_t)ippGetInteger(attr, 0);
-        dbus_message_iter_append_uint32(&iter, &val);
+//        dbus_message_iter_append_uint32(&iter, &val);
       }
       else
 	goto bail;
@@ -490,7 +490,7 @@ main(int  argc,				/* I - Number of command-line args */
       if (attr)
       {
         dbus_uint32_t val = (dbus_uint32_t)ippGetInteger(attr, 0);
-        dbus_message_iter_append_uint32(&iter, &val);
+//        dbus_message_iter_append_uint32(&iter, &val);
       }
       else
 	goto bail;
@@ -515,8 +515,8 @@ main(int  argc,				/* I - Number of command-line args */
 	  strlcpy(p, ippGetString(attr, i, NULL), reasons_length - (size_t)(p - job_reasons));
 	  p += strlen(p);
 	}
-	if (!dbus_message_iter_append_string(&iter, &job_reasons))
-	  goto bail;
+//	if (!dbus_message_iter_append_string(&iter, &job_reasons))
+//	  goto bail;
       }
       else
 	goto bail;
@@ -526,11 +526,11 @@ main(int  argc,				/* I - Number of command-line args */
       if (attr)
       {
         const char *val = ippGetString(attr, 0, NULL);
-        if (!dbus_message_iter_append_string(&iter, &val))
-          goto bail;
+//        if (!dbus_message_iter_append_string(&iter, &val))
+//          goto bail;
       }
-      else
-	dbus_message_iter_append_string(&iter, &nul);
+//      else
+//	dbus_message_iter_append_string(&iter, &nul);
 
       /* UINT32 job-impressions-completed */
       attr = ippFindAttribute(msg, "job-impressions-completed",
@@ -538,14 +538,14 @@ main(int  argc,				/* I - Number of command-line args */
       if (attr)
       {
         dbus_uint32_t val = (dbus_uint32_t)ippGetInteger(attr, 0);
-        dbus_message_iter_append_uint32(&iter, &val);
+//        dbus_message_iter_append_uint32(&iter, &val);
       }
       else
 	goto bail;
     }
 
-    dbus_connection_send(con, message, NULL);
-    dbus_connection_flush(con);
+//    dbus_connection_send(con, message, NULL);
+//    dbus_connection_flush(con);
 
    /*
     * Cleanup...
@@ -553,7 +553,7 @@ main(int  argc,				/* I - Number of command-line args */
 
     bail:
 
-    dbus_message_unref(message);
+//    dbus_message_unref(message);
 
     if (printer_reasons)
       free(printer_reasons);
