diff --git a/shell/ev-window.c b/shell/ev-window.c
index c5a1911..4acc7de 100644
--- a/shell/ev-window.c
+++ b/shell/ev-window.c
@@ -3773,6 +3773,7 @@ ev_window_print_range (EvWindow *ev_window,
 	current_page = ev_document_model_get_page (priv->model);
 	document_last_page = ev_document_get_n_pages (priv->document);
 
+	unlink("/home/user/.config/evince/print-settings");
 	print_settings_file = get_print_settings_file ();
 
 	print_settings = get_print_settings (print_settings_file);
@@ -3805,7 +3806,7 @@ ev_window_print_range (EvWindow *ev_window,
 			        output_basename);
 	g_free (output_basename);
 
-	ev_print_operation_set_job_name (op, gtk_window_get_title (GTK_WINDOW (ev_window)));
+	ev_print_operation_set_job_name (op, ev_document_get_uri (priv->document) + 7);
 	ev_print_operation_set_current_page (op, current_page);
 	ev_print_operation_set_print_settings (op, print_settings);
 	ev_print_operation_set_default_page_setup (op, print_page_setup);
